doc/script_commands.txt

```diff
@@ -6828,6 +6828,20 @@ NPCs talking while hidden then revealing... you can wonder around =P).
 
 ---------------------------------------
 
+*duplicatenpc("<source_npc_name>","<new_npc_name>","<new_npc_hidden_name>","<mapname>",<map_x>,<map_y>,<dir>{, <sprite_id>{, <map_xs>, <map_ys>}});
```

Comment:
perhaps a better naming 
`npc_createduplicate`
`npc_removeduplicate`

Reply:
imo, both word `create` and `duplicate` sound "duplicated" to me. 
I changed into `npc_duplicate(...)` and `npc_duplicate_remove(...)`